src/components/DataTable/TableExpandedRow.js

```diff
@@ -22,17 +22,9 @@ const TableExpandedRow = ({
   const rowRef = useRef(null);
   const className = cx(`${prefix}--expandable-row`, customClassName);
 
-  const onMouseEnter = () => {
-    if (rowRef && rowRef.current) {
-      rowRef.current.previousSibling.classList.add(
-        `${prefix}--expandable-row--hover`
-      );
-    }
-  };
-
-  const onMouseLeave = () => {
-    if (rowRef && rowRef.current) {
-      rowRef.current.previousSibling.classList.remove(
+  const toggleParentHoverClass = () => {
+    if (rowRef && rowRef.current && rowRef.current.previousElementSibling) {
+      rowRef.current.previousElementSibling.classList.toggle(
```

Comment:
The attempt to converge two event listener is good! - Instead of using to args `.toggle()`, we can use three-args `.toggle()` with `evt.type === 'mouseenter'` for the 3rd arg, to make it explicit that we are adding the class on hover.

Reply:
I believe that argument is not supported by IE :/
<img width="298" alt="Screen Shot 2019-04-05 at 6 30 32 PM" src="https://user-images.githubusercontent.com/4078018/55661327-ea771e80-57d0-11e9-97a5-97ab95989eff.png">

Reply:
Oh I thought we had a polyfill for that but it seems false. Good catch - A couple of approaches here:

* Bring in [the polyfill for vanilla](https://github.com/IBM/carbon-components/blob/v10.0.0/demo/polyfills/toggle-class.js) to the [polyfills directory](https://github.com/IBM/carbon-components-react/tree/v7.0.0/.storybook/polyfills) and update [the polyfills list](https://github.com/IBM/carbon-components-react/blob/v7.0.0/.storybook/polyfills.js) and [the migration guide](https://github.com/IBM/carbon-components-react/blob/v7.0.0/docs/migration/migrate-to-7.x.md#polyfills)
* `.classList[evt.type === 'mouseover' ? 'add' : 'remove'](className)`

Reply:
I just went ahead and added an argument to the event handler.